tensorflow/core/kernels/linalg/lu_op_gpu.cu.cc

```diff
@@ -182,9 +182,16 @@ class LuOpGpu : public AsyncOpKernel {
       auto packed_triangular_factors_ptrs = solver->GetScratchSpace<uint8>(
           sizeof(Scalar*) * batch_size, "packed_triangular_factors_ptrs",
           /* on_host */ true);
+#if GOOGLE_CUDA
       const Scalar** packed_triangular_factors_ptrs_base =
           reinterpret_cast<const Scalar**>(
               packed_triangular_factors_ptrs.mutable_data());
+#else //TENSORFLOW_USE_ROCM
+       Scalar** packed_triangular_factors_ptrs_base =
+          reinterpret_cast< Scalar**>(
+              packed_triangular_factors_ptrs.mutable_data());
+#endif
+
```

Comment:
Above changes to get Bazel to stop complaining about unidentified function GetrsBatched. To be explicit, cudaSolvers expects this variable to have the const modifier, whereas rocSolver does not expect const.